charts/argo-cd/templates/argocd-applicationset/clusterrole.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.createClusterRoles }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "argo-cd.applicationSet.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    {{- include "argo-cd.labels" (dict "context" . "component" .Values.applicationSet.name "name" .Values.applicationSet.name) | nindent 4 }}
+rules:
```

Comment:
Can you give us the doc what you refer to define this ? 🙋

Reply:
Copy from https://github.com/argoproj/argo-cd/blob/0e67ed89acac213a0dc358001d1fda2804b2d46d/manifests/base/applicationset-controller/argocd-applicationset-controller-role.yaml#L10-L65.